scarb/src/compiler/compilation_unit.rs

```diff
@@ -24,9 +23,8 @@ pub struct CompilationUnit {
     pub components: Vec<CompilationUnitComponent>,
     /// The profile contains information about *how* the build should be run, including debug
     /// level, etc.
+    // Includes the Cairo compiler configuration parameters to use in this unit.
     pub profile: Profile,
-    /// Cairo compiler configuration parameters to use in this unit.
-    pub compiler_config: ManifestCompilerConfig,
 }
```

Comment:
Hmm, shouldn't compiler_config be modified somewhere up? I currently think `profile` field here should perhaps only keep profile name?